### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by gvan12**
includes some cms work and a _little_ pattern page revamp.

### CODE REVIEW

**Comment by gvan12** [comment]
@jcooter this isn't perfect - but I think it's better than what we currently have - if you're ok with it push once reviewed.

File: src/templates/index-page.js [hunk 1]
```diff
@@ -1,17 +1,53 @@
...
 
 export const IndexPageTemplate = ({ title, content, contentComponent }) => {
   const PageContent = contentComponent || Content;
 
   return (
     <section className="section section--gradient">
+      <div className="centered">
+        <HeroQuestion>I want to...</HeroQuestion>
+        <div className="big-button">
+          <Link to="/request-supplies">Volunteer</Link>
```
Comment (bpugh) [inline src/templates/index-page.js]: This link should be:
```suggestion
          <Link to="/volunteer">Volunteer</Link>
```
shouldn't it?

Comment (gvan12) [reply src/templates/index-page.js]: Yes! Thank you! Will change now!

**Comment by gvan12** [comment]
https://airtable.com/tbl8suD7CTFxg1jlU/viwo7yKkJxnCHpIaq?blocks=hide

New form for requests

**Comment by gvan12** [comment]
https://airtable.com/shrG3kgFmeLYyoxxn/embed

**Pull request merged**